<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GammaRay::AcquisitionWidget</class>
 <widget class="QWidget" name="GammaRay::AcquisitionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>34</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <property name="horizontalSpacing">
    <number>2</number>
   </property>
   <item row="0" column="14">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="17">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Usage</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QToolButton" name="clearButton">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset theme="edit-clear"/>
     </property>
     <property name="iconSize">
      <size>
       <width>15</width>
       <height>15</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QToolButton" name="refreshButton">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset theme="view-refresh"/>
     </property>
     <property name="iconSize">
      <size>
       <width>15</width>
       <height>15</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="9">
    <widget class="QSpinBox" name="bufferSizeSpinBox">
     <property name="suffix">
      <string> MiB</string>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Buffer size:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="18">
    <widget class="QProgressBar" name="usageBar">
     <property name="value">
      <number>25</number>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Sampling rate:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="19">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Overruns:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QToolButton" name="pauseButton">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset theme="media-playback-pause"/>
     </property>
     <property name="iconSize">
      <size>
       <width>15</width>
       <height>15</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QToolButton" name="stopButton">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset theme="media-playback-stop"/>
     </property>
     <property name="iconSize">
      <size>
       <width>15</width>
       <height>15</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="startButton">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset theme="media-playback-start"/>
     </property>
     <property name="iconSize">
      <size>
       <width>15</width>
       <height>15</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="7">
    <widget class="QDoubleSpinBox" name="samplingRateSpinBox">
     <property name="suffix">
      <string> sps</string>
     </property>
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="20">
    <widget class="QLabel" name="overrunLabel">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
